### DISCUSSION TITLE
No Plots

### DISCUSSION THREAD

**Discussion by melvinkeppel** [fmmattioni/lactater#7] [Q&A] [answered]
Hi, i am new to coding, and started with python. I am not able to get the plot shown in the example. Nothing happen when I put this in R and run.

[library](https://rdrr.io/r/base/library.html)(lactater)

demo_data

results_overall <- [lactate_threshold](https://fmmattioni.github.io/lactater/reference/lactate_threshold.html)(
  .data = demo_data, 
  intensity_column = "intensity", 
  lactate_column = "lactate", 
  heart_rate_column = "heart_rate",
  method = [c](https://rdrr.io/r/base/c.html)("Log-log", "OBLA", "Bsln+", "Dmax", "LTP", "LTratio"),
  fit = "3rd degree polynomial", 
  include_baseline = TRUE, 
  sport = "cycling",
  plot = TRUE
)

When I this I see noting, when I run results_overall I get the data in a table.

**Comment by fmmattioni** [comment]
The plots are accessible through the `plot` column in `results_overall`

**Reply by melvinkeppel** [reply]
When I run with the demo_data

```
results_overall <- lactate_threshold(
  .data = demo_data, 
  intensity_column = "intensity", 
  lactate_column = "lactate", 
  heart_rate_column = "heart_rate",
  method = c("Log-log", "OBLA", "Bsln+", "Dmax", "LTP", "LTratio"),
  fit = "3rd degree polynomial", 
  include_baseline = TRUE, 
  sport = "cycling",
  plot = TRUE
)
```

Nothing pops up on a Mac wit R. when I run `print(results_overall)` is see the table. What code should I run to see the graphs?

Can not find the  plot column in results_overall

**Reply by fmmattioni** [reply, marked as answer]
you need to run `results_overall$plot`
